### DISCUSSION TITLE
Long time to reserve a room

### DISCUSSION THREAD

**Discussion by OmegaAhmed** [LibreBooking/librebooking#517] [Q&A] [answered]
How can I make the room reservation process quicker? and didn't take a long time.

**Comment by JohnVillalovos** [comment]
More details would be useful. List the steps you are taking and what part is slow.

**Comment by OmegaAhmed** [comment]
After the user logs in and chooses the room to book, when they click on the create button, it takes time to receive the message that the reservation has been successfully created.

**Comment by servicios-investigacion** [comment, marked as answer]
I have the same issue. I noticed that when I disable email sending, the error disappears.

**Comment by KeonPaul** [comment]
> I have the same issue. I noticed that when I disable email sending, the error disappears.

I have noticed the same issue as well. Any action that causes the generation of an email for example creating, updating or accepting a reservation takes a noticeable amount of time. However once you disable emails, the wait time for those events significantly decreases.
